src/core/security/secure_endpoint.c

```diff
@@ -181,9 +181,9 @@ static void on_read(void *user_data, gpr_slice *slices, size_t nslices,
     return;
   }
   /* The upper level will unref the slices. */
-  input_buffer_count = ep->input_buffer.count;
+  input_buffer_count = (int)ep->input_buffer.count;
```

Comment:
How about making `input_buffer_count` a `size_t` so that it is consistent with the change on `ouput_buffer_count` (see below).